python/paddle/tensor/logic.py

```diff
@@ -372,3 +374,35 @@ def not_equal(x, y, name=None):
     """
     out = fluid.layers.not_equal(x, y, name=name, cond=None)
     return out
+
+
+def is_tensor(x):
+    """
+
+    This function tests whether input object is a paddle.Tensor or a paddle.ComplexTensor.
+
+    Args:
+        x (object): Object to test.
+
+    Returns:
+        Tensor: A boolean value. True if 'x' is a paddle.Tensor or a paddle.ComplexTensor.
```

Comment:
1. Returns "a Tensor" or "a boolean value" ? From code you are returning "a boolean value". Please delete "Tensor:"

2. "True if 'x' is a paddle.Tensor or a paddle.ComplexTensor, otherwise False".
Add "otherwise False"

Reply:
Modified.